### DISCUSSION TITLE
Communication ebusd v5 wlan with home assistant mqtt

### DISCUSSION THREAD

**Discussion by Ziesl** [john30/ebusd#1206] [Q&A]
Hello. 

I am new on this topic. So I am sorry, when I ask stupid questions. I tried to find some explanations on github. But seems like I have problem with the basics. 

I want to do a reading of values of my vailant heating system with ebusd and MQTT (mosquitto) from home assistant. I configured the ebusd adapter with the browser. I set there the WLAN settings of my home WLAN. Now I reach the browser also over my home net. So that part should work. In home assistant I installed eBUSd and configure there the parameters of the ebusd adapter (IP Adress xxx.xxx.xxx.xxx:9999). Here are some additional information:
![ConfigHomeAssistant](https://github.com/john30/ebusd/assets/150350957/9ddfccbc-6dce-40f4-b6b1-b63035cf0dba)
Now I see in sensors form eBUSd in home assistant f. e. running, newest version, version, ... But I see there no additional values from the heating system, f. e. temperatures, etc. (the heating system is connected to the adapter and the LED is lighting green and blinking all 5s blue, so seems like ok). So I see no entities etc. from the heating system. In the log from ebusd in home assistant stands always: "[time stamp] [main error] scan config 15: ERR: wrong symbol received"
When I open the browser of the adapter ebusd says ebusd connected: yes (inactive); eBUS signal: qcquired; So looks like also ok there. 

Now a few questions from me.
Do I have a option to see directly over the browser from the ebusd adapter values from the heating system? In easi> i see not really helpfull commands over the command help. 
Do you have an idea what I have done wrong? Why I see no entities in ha?
Unfortunalety I found not a detailed explanation how I can have access to the ebusd adpter over WLAN (f. e. to configure, or look some values, ...) Maybe you have a detailed explanation of how to do (access to the adapter over WLAN, configuration over WLAN, ...).

Info from ebusd adapter:
Chip ID: dc5475bc05d4, [ESP32-C3, rev 4]
PCB: 24.1
Build: 20240106

Info from eBUSd in ha:
version: 23.2.3
![LogEbusdAdapter](https://github.com/john30/ebusd/assets/150350957/0d976a05-af67-4034-82b7-0a073bcfe371)

Maybe I just found not the wiki entries which I need. Or I did something really wrong. If you need anything else, please just write. It would be a pleasure when you can help me. 

Thank you

**Comment by Koky05** [comment]
Try to look here [Home Assistant auto discovery](https://github.com/john30/ebusd/discussions/518). It was explained how to configure eBusD HA Addon to SSH access and way how you can read date from adapter.

**Reply by Ziesl** [reply]
Okey. I read the whole description there. I found unfortunately nothing about reading values over SSH. I do the configuration in the ha addon ebusd as described there. You can see in the log from the ha addon there:
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
> ebusd --foreground --mqtthost=192.168.xxx.xxx --mqttport=1883 --mqttuser=xxx --mqttpass=xxx --scanconfig --mqttjson --latency=10 --pollinterval=5 --mqttint=/etc/ebusd/mqtt-hassio.cfg --mqtttopic=ebusd --device=192.168.xxx.xxx:9999 --log=all:notice --log=main:notice --log=other:notice
2024-02-27 19:36:15.897 [main notice] ebusd 23.2.p20230716 started with auto scan on device: 192.168.xxx.xxx5:9999
2024-02-27 19:36:16.607 [bus notice] bus started with own address 31/36
2024-02-27 19:36:16.622 [mqtt notice] connection established
2024-02-27 19:36:16.634 [bus notice] signal acquired
2024-02-27 19:36:23.004 [bus notice] max. symbols per second: 118
2024-02-27 19:36:43.015 [bus notice] max. symbols per second: 133
2024-02-27 19:37:04.005 [bus notice] max. symbols per second: 136
2024-02-27 19:37:55.035 [bus notice] max. symbols per second: 156
2024-02-27 19:38:26.793 [main notice] update check: version 23.3 available
2024-02-27 19:40:56.036 [bus notice] max. symbols per second: 157
2024-02-27 19:47:36.035 [bus error] signal lost
2024-02-27 19:47:43.782 [bus notice] device invalid
2024-02-27 19:47:48.815 [bus notice] re-opened 192.168.xxx.xxx:9999
2024-02-27 19:47:48.843 [bus notice] signal acquired

So it says connected. But I see no additonal values (entities), as I described in my topic above your answer. 

I still do not know what I did wrong there. I have several sensors over MQTT in my HA. Is working fine there. So the broker is set correctly there, otherwise it would be not possible to see values over MQTT.

**Reply by Koky05** [reply]
In addon you need to specify port to open SSH canal. Then you can connect via IP address of HA and specified port.
Or if you have installed also SSH Terminal as Addon you can use command:

> docker exec -it `docker ps | grep ebusd | awk '{print $1}'` /bin/bash

In shell you can call standard ebusctl command to read values [TCP client commands](https://github.com/john30/ebusd/wiki/3.1.-TCP-client-commands). For example `ebusctl i` to show identified circuit and loaded configuration files.
